### PR TITLE
Nc news 3

### PR DESCRIPTION
**Pull request by jimmybombimmy**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -33,4 +39,81 @@ describe("GET /api/topics", () => {
...
+      return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({
+          body
+        }) => {
+          expect(body).toEqual(endpoints)
+        })
+    })
+    test('200: each object in endpoints file should return with description, query, body format and an example response (except for the first api example)', () => {
```
Comment (RichardNeat) [inline __tests__/api.test.js:80]: Technically we don't need to worry about checking for the individual content keys etc in this test, as previously we have required the file in and checked the response matches the entire contents of that file. So we have checked the whole thing already.

File: __tests__/api.test.js [hunk 2]
```diff
@@ -33,4 +39,81 @@ describe("GET /api/topics", () => {
...
   })
-})
\ No newline at end of file
+}),
+
+describe('GET api/articles', () => {
+
+}),
+
+// Test 1 - api page returns with an object
```
Comment (RichardNeat) [inline __tests__/api.test.js:48]: Please remove all comments and console logs before committing 👍

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice work James - I've left a few comments below for requested changes please. If you have any questions just drop me a message. Once you've made those changes, you can go ahead and merge :)

File: __tests__/api.test.js (hunk 2 above)
Comment (jimmybombimmy) [reply __tests__/api.test.js:48]: > Please remove all comments and console logs before committing 👍

Would it be good practice to leave the comments in explaining how the tests for the length is working? I understand deleting the test 1 etc. but am just curious as to whether I should keep this bit in or not.

**Pull request merged**